multiversx_sdk/transfers/transfers_controller_test.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from multiversx_sdk.accounts.account import Account
+from multiversx_sdk.core.address import Address
+from multiversx_sdk.core.tokens import Token, TokenTransfer
+from multiversx_sdk.entrypoints.entrypoints import DevnetEntrypoint
+from multiversx_sdk.wallet.user_keys import UserSecretKey
+
+
+class TestTransfersController:
+    sender = Account(UserSecretKey(bytes.fromhex("bdf3c95c4b0bcbacd828b148231a10c25f93286befe92077b5d096055fb4e96a")))
+    mike = Address.new_from_bech32("erd1uv40ahysflse896x4ktnh6ecx43u7cmy9wnxnvcyp7deg299a4sq6vaywa")
+    grace = Account.new_from_pem(Path(__file__).parent.parent / "testutils" / "testwallets" / "grace.pem")
+    usdc = "USDC-350c4e"
+
+    entrypoint = DevnetEntrypoint(kind="proxy", with_gas_limit_estimator=True)
```

Comment:
I remember we use a specific pytest marker for devnet tests.

Reply:
added